### PR TITLE
Adds velocity LSQ solution test

### PR DESCRIPTION
**Pull request by jtec**
This PR adds a unit test computing antenna velocity using Doppler observations, thereby closing https://github.com/jtec/prx/issues/42.

### CODE REVIEW

**Comment by jtec** [comment]
@plutonheaven I noticed that Doppler observations in the CSV are in Hz and not converted to m/s. I wonder whether that is not the more appropriate choice, `prx` being just a preprepocessor, and Doppler obs in Hz being the "natural" observation of a receiver?

**Comment by plutonheaven** [comment]
> @plutonheaven I noticed that Doppler observations in the CSV are in Hz and not converted to m/s. [...]

I think both thing would make sense, converting everything to m or m/s, or keeping the original units of the RINEX format.

I would tend to keep the RINEX units, because converting Doppler from Hz to m/s makes some assumptions on the smaller satellite clock drift and on the user position (for LOS vector computation). Notably, if LEO PNT really emerge, those assumptions may change (larger Doppler effect, smaller distance, cheaper satellite clocks with larger drift ???)

It could better to leave those conversion to the user, so that he makes a conscious choice about it.

So let us keep Doppler obs in Hz and carrier obs in cycles!

File: src/prx/user.py [hunk 1]
```diff
@@ -15,7 +16,38 @@ def parse_prx_csv_file(prx_file: Path):
     return pd.read_csv(prx_file, comment="#"), parse_prx_csv_file_metadata(prx_file)
 
 
+def spp_vt_lsq(df, p_ecef_m):
+    df = df[df.D_obs_hz.notna()].reset_index(drop=True)
+    df["D_obs_mps"] = df.D_obs_hz * cGpsSpeedOfLight_mps / df.carrier_frequency_hz
```
Comment (plutonheaven) [inline src/prx/user.py:21]: According to https://gnss-sdr.org/docs/sp-blocks/observables/#doppler-shift-measurement, the conversion to mps involves a minus sign.

`df["D_obs_mps"] = df.D_obs_hz * cGpsSpeedOfLight_mps / df.carrier_frequency_hz`

Then, the corrected pseudorange rate would have the following signs:
`df["D_obs_corrected_mps"] = (
        df.D_obs_mps.to_numpy().reshape(-1, 1)
        + df.dclock_mps.to_numpy().reshape(-1, 1)
        - df["satellite_los_velocities"].to_numpy().reshape(-1, 1)
    )`

All those sign errors are unobservable for fixed data collect.

We should probably put a short dynamic trajectory and verify if the estimated velocity using the doppler observations corresponds to the time difference of the estimated position.

Comment (jtec) [reply src/prx/user.py:21]: Totally agree, added https://github.com/jtec/prx/issues/66

Comment (jtec) [reply src/prx/user.py:21]: Thanks for checking the sign, made the change.

**Review by plutonheaven — APPROVED** [review]

**Pull request merged**